### PR TITLE
Fix some nullable return annotations 2

### PR DESCRIPTION
**Pull request by sophie-h**
Issue #68

### CODE REVIEW

File: gtk4/src/auto/header_bar.rs [hunk 1]
```diff
@@ -36,7 +36,7 @@ impl HeaderBar {
         unsafe { Widget::from_glib_none(ffi::gtk_header_bar_new()).unsafe_cast() }
     }
 
-    pub fn get_decoration_layout(&self) -> Option<glib::GString> {
+    pub fn get_decoration_layout(&self) -> glib::GString {
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/header_bar.rs:39]: This should be an option per your upstream patch

**Comment by bilelmoussaoui** [comment]
looks good to me otherwise, do you want to exchange reviews with me on #83 ? :)

File: gtk4/src/auto/header_bar.rs (hunk 1 above)
Comment (sophie-h) [reply gtk4/src/auto/header_bar.rs]: True, upstream is correct. Thanks!

**Comment by bilelmoussaoui** [comment]
Thanks!

**Comment by sophie-h** [comment]
Sorry @bilelmoussaoui didn't had time for a review. Since upstream didn't merged your complete MR, does this need some kind of update?

**Pull request merged**